### PR TITLE
Add aggregate-anomaly to Total Anomaly plot

### LINKED ISSUE #848: Add anomaly thresholds to Grafana dasboard(s)
Would be cool if we could at least show the calculated aggregated anomaly threshold in the total anomaly panel.

Postgres: `select jsonb_each.value as aggregate_threshold from machine, jsonb_each(machine.metadata->'build_metadata'->'model'->'model_meta') where name='$Machine' and key='aggregate-threshold'`

### PR DESCRIPTION
**Pull request by milesgranger**
Will close #848 

Adds a static plotting of the `aggregate-anomaly` calculated in the `DiffBasedAnomalyDetector` and plots it onto the existing _"TotalAnomaly"_ panel:

image

### CODE REVIEW

File: resources/grafana/dashboards/machines.json [hunk 1]
```diff
@@ -966,6 +971,33 @@
...
               }
             ]
+          },
+          {
+            "datasource": "Postgres-metadata",
+            "format": "time_series",
+            "group": [],
+            "metricColumn": "none",
+            "rawQuery": true,
+            "rawSql": "select \n  cast (threashold.value as float) as threshold, \n  date(machine.dataset->>'train_start_date') as time \nfrom machine, jsonb_each(machine.metadata->'build_metadata'->'model'->'model_meta') as threashold \nwhere name='$Machine' and key='aggregate-threshold'\n\nunion\n\nselect \n  cast (threashold.value as float) as threshold, \n  current_date as time \nfrom machine, jsonb_each(machine.metadata->'build_metadata'->'model'->'model_meta') as threashold \nwhere name='$Machine' and key='aggregate-threshold'",
```
Comment (flikka) [inline resources/grafana/dashboards/machines.json]: typo "threashold"

File: resources/grafana/dashboards/machines.json [hunk 2]
```diff
@@ -941,8 +946,8 @@
             "measurement": "total-anomaly-scaled",
             "orderByTime": "ASC",
             "policy": "default",
-            "query": "SELECT mean(/^$Sensor$/) as \"field\" FROM \"original-input\" WHERE (\"machine\" =~ /^$Machine$/) AND $timeFilter GROUP BY time($__interval) fill(null)",
-            "rawQuery": false,
```
Comment (flikka) [inline resources/grafana/dashboards/machines.json:945]: weird that this ever worked...

Comment (milesgranger) [reply resources/grafana/dashboards/machines.json:945]: :man_shrugging: Ran it though the test project and it works for sure now at least.

**Comment by milesgranger** [comment]
commit https://github.com/equinor/gordo/pull/850/commits/f48254bae3b052cbf4e643334266da20dc1de3f1 changes the metrics panel to show all metrics:

image

Can drop this if desired, otherwise I think it's cool to see all the metrics in one place.

**Review by flikka — APPROVED** [review]
Cool! Would be awesome to have this in a release today (or monday morning)

**Pull request merged**